deployments/ansible/roles/collector/tasks/win_fluentd_install.yml

```diff
@@ -22,3 +22,29 @@
     dest: "{{splunk_fluentd_config}}"
   when: splunk_fluentd_config_source != ""
   notify: "restart windows fluentdwinsvc"
+
+- name: Copy fluentd default config
+  ansible.windows.win_copy:
+    src: "{{splunk_fluentd_config}}"
+    dest: "{{ansible_env.SystemDrive}}/opt/td-agent/etc/td-agent/td-agent.conf"
```

Comment:
Don't we use `\` across the ansible jobs for windows?

Reply:
Okay.